lib/refresher.ts

```diff
@@ -11,18 +11,16 @@ class Refresher {
   }
 
   public authorize = (request: Function): Promise<any> => {
-    const self = this
+    const self = this;
```

Comment:
@gilmcquillan I suppose the first line `import 'isomorphic-fetch'` is **redundant** here, am I right?